### ISSUE TITLE
Osei-b4: Headless-Cms: 3 week

### ISSUE THREAD

**Issue by Osei-b4** [HYF-Class20/home#165]
# Learning - Objectives 

- [ ] 🥚 Understanding what a content management system is, and why people use it 
- [ ] 🥚 Given an existing CMS installation fill in and publish a new piece of content
- [ ] 🥚 Perform a simple http call via postman to retrieve a piece of information.
- [ ] 🐣 Extend and adapt an existing CMS configuration
- [ ] 🐣 Perform a http call with filtering via Postman
- [ ] 🐣 Perform a http call with "include" via Postman
- [ ] 🐣 Perform a http POST call to create new data
- [ ] 🐣 Perform a http PUT call to update existing data
- [ ] 🐣 Configuring endpoint permissions (Authentication vs Authorization)
- [ ] 🐥 Design a data model from scratch based on user requirements

- I have practiced:
  - [x] pulling and pushing
  - [x] formatting my code: `npm run format`
  - [x] checking my spelling: `npm run spell-check`
  - [x] linting my code: `npm run lint:ls` _and_ `npm run lint:md`
  

## Week 1

- [x] I have pushed my progress to [my fork of exercises repo](https://github.com/Osei-b4/headless-cms/tree/react-dome)

## I Need Help With:
How to engage with some `REACT COMPONENTS`

## What went well?
How `react` works around the on `JavaScript`

## What went less well?
nothing yet

**Comment by bermarte** [comment]
@Osei-b4 you are still in the React Module.

**Comment by Osei-b4** [comment]
@bermarte Yes please i had a laptop problem that is why, but I will catch up soon
